student_class.rb

```diff
@@ -0,0 +1,13 @@
+require './Person_Class'
+
+# This is the teacher class inherit from person
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] [REQUIRED] Please give correct value to in line 5, it would like this ``def initialize(age, classroom, name = "Unknown", parent_permission = true)``